src/components/editor/InputLiteral.jsx

```diff
@@ -98,27 +116,78 @@ export class InputLiteral extends Component {
      }
   }
 
+  dispModal (content) {
+    return(
+      <Modal show={this.state.show} onHide={this.handleClose}>
+        <Modal.Header closeButton>
+          <Modal.Title>Languages</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <InputLang textValue={content}/> 
+        </Modal.Body>
+        <Modal.Footer>
+          <Button onClick={this.handleClose}>Submit</Button>
+          <Button onClick={this.handleClose}>Close</Button>
+        </Modal.Footer>
+      </Modal>
+    )
+  }
+
+  dispLang(content){
+    let newState = store.getState()
+    var index = newState.lang.formData.map(function(o) { return o.id; }).indexOf(content);
+    var newLang;
+    try {
+      newLang = newState.lang.formData[index].items[0].label
+    } catch (error) {
+      // ignore
+    }
+
+    if (newLang == undefined) {
+      return "English"
+    } else{
+      return newLang
+    }
+  }
+
+  // Bug
+  // When adding the same value twice to the input field, the languages concat and then we have a problem.
+  // English is the default value, but is not set in the redux.lang.store. Needs to be set manually in the
+  // generation of RDF. 
```

Comment:
are there tickets for the two issues above?  and for the one below?

Reply:
creating them now

Reply:
put tix # into comments?  Or is that too picky?